modules/server_gcs/main.tf

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#--------#
+# Locals #
+#--------#
+locals {
+  random_hash             = var.suffix
+  storage_bucket_name     = "forseti-server-${local.random_hash}"
```

Comment:
This variable does not seem to be used, and it looks like it's the same as `server_bucket_name`.